rosa_tutorials/rosa-mobb-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,336 @@
+----
+$ cat << EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  upgradeStrategy: Default
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: aws-load-balancer-operator
+  namespace: aws-load-balancer-operator
+spec:
+  channel: stable-v1.0
+  installPlanApproval: Automatic
+  name: aws-load-balancer-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: aws-load-balancer-operator.v1.0.0
+EOF
+----
++
+. Install Red Hat AWS Load Balancer Controller:
++
+[NOTE]
+====
+If you get an error here wait a minute and try again, it means the Operator has not completed installing yet.
+====
++
+[source,terminal]
+----
+$ cat << EOF | oc apply -f -
+apiVersion: networking.olm.openshift.io/v1
+kind: AWSLoadBalancerController
+metadata:
+  name: cluster
+spec:
+  credentials:
+    name: aws-load-balancer-operator
+EOF
+----   
++
+. Check the Operator and controller pods are both running:
++
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator get pods
+----
++
+You should see the following, if not wait a moment and retry:
++
+[source,terminal]
+----
+NAME                                                             READY   STATUS    RESTARTS   AGE
+aws-load-balancer-controller-cluster-6ddf658785-pdp5d            1/1     Running   0          99s
+aws-load-balancer-operator-controller-manager-577d9ffcb9-w6zqn   2/2     Running   0          2m4s
+----
+
+== Validate the deployment with Echo Server application
+
+. Deploy Echo Server Ingress with ALB: 
++
+[source,terminal]
+----
+$ oc new-app -n hello-world --image=docker.io/openshift/hello-openshift
+----
++
+. Curl the ALB ingress endpoint to verify the echoserver pod is accessible: 
++
+[source,termnial]
+----
+$ INGRESS=$(oc -n echoserver get ingress echoserver \
+    -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+  curl -sH "Host: echoserver.example.com" \
+    "http://${INGRESS}" | grep Hostname
+
+  Hostname: echoserver-7757d5ff4d-ftvf2
+----
++
+. Deploy Echo Server NLB Load Balancer:
++
+[source,terminal]
+----
+$ cat << EOF | oc apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: echoserver-nlb
+  namespace: echoserver
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-type: external
+    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: instance
+    service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+spec:
+  ports:
+    - port: 80
+      targetPort: 8080
+      protocol: TCP
+  type: LoadBalancer
+  selector:
+    app: echoserver
+EOF
+----
++
+. Test the NLB endpoint:
++
+[source,terminal]
+----
+$ NLB=$(oc -n echoserver get service echoserver-nlb \
+  -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+ curl -s "http://${NLB}" | grep Hostname
+
+ Hostname: echoserver-7757d5ff4d-ftvf2
+----
+
+== Clean up
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/122639474/5d5c2fdf-fc58-4be5-bb8f-70d99a48ef51)

Reply:
FWIW, This is what I had for clean up of ALBO, etc.